tests/setup.py

```diff
@@ -250,6 +250,12 @@ def grant_permission_to_service_account(self, permission, argument, service_acco
         )
         grant.add(self.session)
 
+    def disable_user(self, user):
+        # type: (str) -> None
+        user_obj = User.get(self.session, name=user)
```

Comment:
Since the user repo/service already exists, feels better to call into the user service `disable_user` function (and similarly for a bunch of the other setup functions that are now able to call into services instead of reimplementing logic)

Reply:
Good catch, I missed that we already had a service call for that. (I think I made the other setup functions use a service or repository if one exists. When I have some time, I'll go through and move that code into repositories systematically.)